lemur/plugins/lemur_gcp/plugin.py

```diff
@@ -75,54 +73,130 @@ def _insert_gcp_certificate(self, project_id, ssl_certificate_body, credentials)
             project=project_id, ssl_certificate_resource=ssl_certificate_body
         )
 
-    def _get_gcp_credentials(self, options):
-        if self.get_option('authenticationMethod', options) == "vault":
-            # make a request to vault for GCP token
-            return self._get_gcp_credentials_from_vault(options)
-        elif self.get_option('authenticationMethod', options) == "serviceAccountToken":
-            if self.get_option('serviceAccountTokenPath', options) is not None:
-                return service_account.Credentials.from_service_account_file(
-                    self.get_option('serviceAccountTokenPath', options)
-                )
+    def _full_ca(self, body, cert_chain):
```

Comment:
Personally, I think it makes sense for both ` _full_ca()` and  `_insert_gcp_certificate()` to be moved to the certificate.py file, since these methods are concerned with performing actions specifically on Certificates.